examples/gno.land/p/demo/airdrop/merkle-airdrop.gno

```diff
@@ -0,0 +1,88 @@
+package airdrop
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/merkle"
+)
+
+var (
+	ErrAlreadyClaimed = errors.New("already claimed")
+	ErrInvalidProof   = errors.New("invalid merkle proof")
+)
+
+type AirdropData struct {
+	Address std.Address
+	// TODO: use std.Coin
+	Amount uint64
```

Comment:
I think you should unexport the attributes

Reply:
For better control and data integrity, keep the fields in the `Airdrop` struct unexported. 


To maintain easy marshaling with the `Bytes` helper, consider this approach:

```go
type Airdrop struct {
    data struct {
        Address std.Address
        Amount  uint64
    }
}
```